mne/viz/raw.py

```diff
@@ -510,6 +510,7 @@ def _prepare_mne_browse_raw(params, title, bgcolor, color, bad_color, inds,
     """Helper for setting up the mne_browse_raw window."""
     import matplotlib.pyplot as plt
     import matplotlib as mpl
+    plt.ion()
```

Comment:
I think it was ``QCoreApplication::exec: The event loop is already running``. I'm not at the office at the moment so I can't check.
Would it be okay to turn it on when the user sets ``block=True`` in params?